Content/default/tests/Client/Client.Tests.fs

```diff
@@ -5,27 +5,29 @@ open Fable.Mocha
 open Index
 open Shared
 
-let client = testList "Client" [
-    testCase "Added todo" <| fun _ ->
-        let newTodo = Todo.create "new todo"
-        let model, _ = init ()
+let client =
+    testList
+        "Client"
+        [ testCase "Added todo"
+          <| fun _ ->
```

Comment:
this looks a bit weird. In fact I'm not a fan of the `<|` operator anyway. Maybe you can suggest a different syntax here?